### PR TITLE
configuration of monitoring; adding mirror ports - with errors.

### PR DESCRIPTION
**Pull request by rasmim**
There is an error in the function that is creating the mirroring ports. 

To be fixed later. 

```
2021/05/15 14:20:27 ovs: exec: sudo [ovs-vsctl --may-exist add-port game getALLblue]
2021/05/15 14:20:27 ovs: exec: "ovs-vsctl: Error detected while setting up 'getALLblue': could not open network device getALLblue (No such device).  See ovs-vswitchd log for details.\novs-vsctl: The default log directory is \"/var/log/openvswitch\"."
2:20PM INF AddPort for mirroring Set Interface Options getALLblue
2021/05/15 14:20:27 ovs: exec: sudo [ovs-vsctl set interface getALLblue type=internal]
2021/05/15 14:20:27 ovs: exec: ""
2021/05/15 14:20:27 ovs: exec: sudo [ovs-vsctl --id=@getALLblue get port getALLblue -- set mirror blueMirror select_all=true select_vlan=10,20,30 output-port=@getALLblue]
2021/05/15 14:20:27 ovs: exec: "ovs-vsctl: unknown command 'get port'; use --help for help"
2:20PM ERR Error on adding port to mirror traffic error="exit status 1: ovs-vsctl: unknown command 'get port'; use --help for help"
2:20PM ERR Error to configure monitoring error="exit status 1: ovs-vsctl: unknown command 'get port'; use --help for help"
```

### CODE REVIEW

File: game/environment.go [hunk 1]
```diff
@@ -166,15 +186,22 @@ func (g *environment) StartGame(tag, name string, scenarioNo int) error {
...
+	}
+
+	if err := g.configureMonitor(bridgeName, numNetworks); err != nil {
+		log.Error().Err(err).Msgf("Error to configure monitoring")
 	}
 
 	if err := g.initializeScenarios(bridgeName, &g.controller, scenarioNo); err != nil {
-		return err
+		log.Error().Err(err).Msgf("Error on initializing scenarios")
 	}
```
Comment (mrtrkmn) [inline game/environment.go:203]: No return err ? Is there specific reason for it ?

**Review by mrtrkmn — COMMENTED** [review]
I have seen a place where it does not return err which it should and proceed without it. In normal cases, it should return the error back to user since the error is directly related to initializing scenarios which need to be created, up and running...
There might be a reason for removing `return err ` if so, it would be great to hear why.

File: game/environment.go (hunk 1 above)
Comment (rasmim) [reply game/environment.go:203]: From my best understanding, the error is returned here: 
`log.Error().Err(err).Msgf("Error on initializing scenarios")`

I have seen when running the code that I get the error message. @Lanestolen Can you comment here, it was your advice :)

Comment (Lanestolen) [reply game/environment.go:203]: Hey, no the error isn't return, how ever the error will be shown as a sperate field in the output json

**Review by mrtrkmn — APPROVED** [review]

**Pull request merged**